### PR TITLE
fix(2497): Add alt attributes to images

### PR DESCRIPTION
**Pull request by tkyi**
## Context

It would be nice to have alt attributes for those that cannot see images in our guide.

## Objective

This PR adds alt attributes to images on our main pages.

## References

Related to https://github.com/screwdriver-cd/screwdriver/issues/2497

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by jithine — APPROVED** [review]
@kumada626 please review the Japanese alt text. Can be addressed in a later PR.

File: docs/ja/about/index.md [hunk 1]
```diff
@@ -20,15 +20,15 @@ toc:
         <p>Screwdriverはあなたのビルドパイプライン上で継続的デリバリーを第一級オブジェクトとして扱います。プルリクエストからプロダクションまでの流れを簡単に定義します。</p>
     </div>
     <div class="col-xs-12 col-md-4">
-        <img src="/assets/continuous_delivery.png" class="cd">
+        <img src="/assets/continuous_delivery.png" class="cd" alt="build publish deploy flowchart">
```
Comment (kumada626) [inline docs/ja/about/index.md]:
```suggestion
        <img src="/assets/continuous_delivery.png" class="cd" alt="ビルド、パブリッシュ、デプロイのフローチャート">
```

File: docs/ja/about/index.md [hunk 2]
```diff
@@ -20,15 +20,15 @@ toc:
...
     </div>
 </div>
 
 
 
 <div class="row">
     <div class="col-xs-12 col-md-4">
-        <img src="/assets/daily_habits.png" class="dh">
+        <img src="/assets/daily_habits.png" class="dh" alt="commit test release cycle">
```
Comment (kumada626) [inline docs/ja/about/index.md]:
```suggestion
        <img src="/assets/daily_habits.png" class="dh" alt="コミット、テスト、リリースのサイクル">
```

File: docs/ja/about/index.md [hunk 3]
```diff
@@ -44,15 +44,15 @@ toc:
         <p>シンプルなYAMLファイルをコードに追加することでパイプラインを定義できます。パイプラインを扱う他の設定は無いため、他のコードと合わせてパイプラインの変更をレビューした上で投入できます。</p>
     </div>
     <div class="col-xs-12 col-md-4">
-        <img src="/assets/pipeline_code.png" class="pc">
+        <img src="/assets/pipeline_code.png" class="pc" alt="Screwdriver.yaml screenshot">
```
Comment (kumada626) [inline docs/ja/about/index.md]:
```suggestion
        <img src="/assets/pipeline_code.png" class="pc" alt="screwdriver.yaml のスクリーンショット">
```

File: docs/ja/about/index.md [hunk 4]
```diff
@@ -44,15 +44,15 @@ toc:
...
     </div>
 </div>
 
 
 
 <div class="row">
     <div class="col-xs-12 col-md-4">
-        <img src="/assets/3rd_party_services.png" class="party">
+        <img src="/assets/3rd_party_services.png" class="party" alt="third party services examples">
```
Comment (kumada626) [inline docs/ja/about/index.md]:
```suggestion
        <img src="/assets/3rd_party_services.png" class="party" alt="対応するサードパーティーサービスの例">
```

File: docs/ja/index.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ menu: menu_ja
 is_homepage: 'true'
 ---
 <div class="top">
-    <img src="/assets/Screwdriver_Icon_Full@1x.png">
+    <img src="/assets/Screwdriver_Icon_Full@1x.png" alt="Screwdriver icon">
```
Comment (kumada626) [inline docs/ja/index.md]:
```suggestion
    <img src="/assets/Screwdriver_Icon_Full@1x.png" alt="Screwdriverのアイコン">
```

**Review by kumada626 — COMMENTED** [review]
I translated ja page alt text in Japanese.

**Review by kumada626 — APPROVED** [review]

**Pull request merged**